pygeo/pyBlock.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, initType, fileName=None, FFD=False, symmPlane=None, **kwargs)
     #                     Initialization Types
     # ----------------------------------------------------------------------
 
-    def _readPlot3D(self, fileName, order="f", FFD=False, symmTol=0.001):
+    def _readPlot3D(self, fileName, order="f", FFD=False, symmTol=0.001, kmax=4):
```

Comment:
I'm guessing this is one of the `kwargs` that gets passed by the `DVGeometry` constructor call? In that case, please document this option both here, and in the `DVGeometry` init call.

Reply:
I guess this is actually passed through the `pyBlock` constructor as well. I don't like undocumented `kwargs` so we should try to do a better job here.

Reply:
tried to clarify how this is done a bit by exposing the kmax in the dvgeo init and the pyblock init. please take a look and let me know if its okay now